core/system_metric/sys_metric_stat.go

```diff
@@ -146,3 +235,17 @@ func CurrentCpuUsage() float64 {
 func SetSystemCpuUsage(cpuUsage float64) {
 	currentCpuUsage.Store(cpuUsage)
 }
+
+func CurrentMemoryUsage() int64 {
+	bytes, ok := currentMemoryUsage.Load().(int64)
+	if !ok {
+		return NotRetrievedMemoryValue
```

Comment:
Maybe we should check if we can retrieve when we loadRules

Reply:
User need to guarantee open memory collector